config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -165,3 +165,55 @@ periodics:
           # this is mostly for building kubernetes
           memory: 14Gi
           cpu: 7
+- interval: 1h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions-feature-gate-test-n-minus-2
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-versions-feature-gate-test
+    description: Uses kind to run bespoke feature gate tests from the n-2 kubernetes release yaml files against a latest kubernetes master components w/ --emulated-version=n-1 set.
+    # TODO(#34269) route the alert email to a rotation vs individual email and update owners in experiment/compatibility-versions
+    testgrid-alert-email: aprindle@google.com
```

Comment:
This should use the alias:
compatibility-version-test@google.com

Related PR: https://github.com/kubernetes/test-infra/pull/34795

Reply:
Since we can't enable the test yet this was removed. Will keep in mind when adding n-2 tests here.